explainer.md

```diff
@@ -0,0 +1,76 @@
+# Screen Capture Explained
+
+## What is it? What’s it for?
+
+It’s an API to get a live video stream of your computer screen or a single window on it, optionally with sound.
+
+It’s sometimes referred to it as screen-sharing, even though it’s only the capturing part being defined (sharing is just the primary use case).
+
+Have you ever been a presenter in an online video conference call? You probably shared your screen or the window of a presentation-program to do that. Have you ever presented in an online video conference call without this feature? That would probably require printouts and hand-waving! It’s a pretty key feature. This is that feature.
+
+
+ * Not the networking part—that’s WebRTC’s RTCPeerConnection
```

Comment:
link for WebRTC (generally, I think that it's a good idea to link to these major components; I don't know if the spec or the repo or the explainer is best, but I'll let you decide)